### PR TITLE
feature/header

### PR DESCRIPTION
**Pull request by HoangHieuHaTran**

### CODE REVIEW

File: sass/style.css [hunk 1]
```diff
@@ -1,5 +1,171 @@
...
+    --purple: #737d9e;
+}
+
+img {
+    max-width: 100%;
+    width: 100%;
+}
+
+.container {
+    max-width: 1170px;
```
Comment (nvanha) [inline sass/style.css:29]: padding left and right is 10px, so max width equals 1170px + 10px + 10px

Comment (harryngdev) [inline sass/style.css:29]: padding left and right is 10px, so max width equals 1170px + 10px + 10px

File: sass/style.css [hunk 2]
```diff
@@ -1,5 +1,171 @@
...
+}
+.navbar {
+    display: flex;
+    padding: 30px;
+    justify-content: space-between;
+    align-items: center;
+}
+/* logo */
+.navbar__logo img{
+    margin-left: 137px;
```
Comment (harryngdev) [inline sass/style.css:48]: margin-left? img not vertically centered?

image

File: sass/style.css [hunk 3]
```diff
@@ -1,5 +1,171 @@
...
+        font-family: "Roboto", sans-serif;
+}
 :root {
-    background : white;
-}
\ No newline at end of file
+    --white: #fff;
+    --smooth: all .3s ease-in;
+    --ticket: #ffdd7b;
+    --blue: #283869;
```
Comment (harryngdev) [inline sass/style.css:17]: wrong color code
image

File: sass/style.css [hunk 4]
```diff
@@ -1,5 +1,171 @@
...
+    text-align: center;
+    background-color: var(--blue); 
+    padding: .8rem 2.5rem;
+    transition: var(--smooth); 
+}
+button :hover {
+ background-color: var(--blue);
+ transition: var(--smooth);
+}
+.navbar__btn {
```
Comment (harryngdev) [inline sass/style.css:99]: wrong
image

File: sass/style.css [hunk 5]
```diff
@@ -1,5 +1,171 @@
...
+    transform: translateX(0px);
+    background: transparent;
+}
+.open .navbar__icon:before{
+    transform: rotate(45deg) translate(35px,-35px);
+}
+.open .navbar__icon:after{
+    transform: rotate(-45deg) translate(35px,35px);
+}
+@media (min-width: 62em) {
```
Comment (harryngdev) [inline sass/style.css:152]: check it again
image

File: index.html [hunk 6]
```diff
@@ -5,15 +5,52 @@
...
-      <li>hello</li>
-      <li>hello</li>
-      <li>hello</li>
-    </ul>
+    <!-- header -->
+    <header>
+      <div class="container">
+        <nav class="navbar">
+          <!-- logo -->
+          <div class="navbar__logo">
```
Comment (harryngdev) [inline index.html:33]: the logo usually be tag <a /> with href='/', to direct back the homepage

File: index.html [hunk 7]
```diff
@@ -5,15 +5,52 @@
...
+            <img src="./img/logo.png" alt="" website logo />
+          </div>
+          <!-- links -->
+          <ul class="navbar__links">
+            <li class="navbar__link">
+              <a href="#"
+                >Events<i class="fa-sharp fa-solid fa-chevron-down"></i
```
Comment (harryngdev) [inline index.html:40]: Missing margin-left in tag <i />

**Pull request closed without merging**